ui/desktop/src/main.ts

```diff
@@ -61,6 +60,7 @@ let appConfig = {
   GOOSE_API_HOST: 'http://127.0.0.1',
   GOOSE_SERVER__PORT: 0,
   GOOSE_WORKING_DIR: '',
+  GOOSE_AGENT_VERSION: '',
```

Comment:
where is this set? (BTW this section can be dynamicaly replaced at build time when packaging it up)

Reply:
(ie in: https://github.com/block/goose/blob/v1.0/ui/desktop/bundle.py)

Reply:
currently not set, but defaults to the "base" version of the app (from goose-server), which is what we want. There's probably a better way to do and happy for feedback on that (first time on the UI side)